app/src/main/java/com/codesoom/assignment/controllers/UserController.java

```diff
@@ -0,0 +1,57 @@
+package com.codesoom.assignment.controllers;
+
+
+import com.codesoom.assignment.application.UserService;
+import com.codesoom.assignment.dto.UserData;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+/**
+ * 사용자 등록, 수정, 삭제에 대한 Request, Response 처리
```

Comment:
Request, Response 처리에 대해 우리는 이렇게 표현할 수 있겠죠.

```suggestion
 * 사용자 정보에 대한 http 처리를 담당합니다.
```

Reply:
네 좀더 간결한 표현인것같습니다.